assets/javascripts/discourse-assign/controllers/group-assigned.js.es6

```diff
@@ -12,6 +12,30 @@ export default Controller.extend({
   filterName: "",
   filter: "",
 
+  @discourseComputed("router.currentRoute.queryParams.order")
```

Comment:
I guess this needs to be changed since the router is gone?

Reply:
we are getting router in the controller [here](https://github.com/discourse/discourse-assign/blob/83ff23c98ec8d5edcb3090093c4025db1d618ef0/assets/javascripts/discourse-assign/controllers/group-assigned.js.es6#L8) we still need to change this?

Reply:
Hmm I see... It is much better now that it's in the controller, but still, injecting the router service is not ideal.

What happens if we add order/ascending/q as queryParams to this controller? Does it break the group-assigned-show controller? Or does it work ok?

Reply:
It breaks the `group-assigned-show` controller

Reply:
Ok, let's keep this then :+1: